CalibTracker/SiPixelConnectivity/src/SiPixelFedCablingMapBuilder.cc

```diff
@@ -30,34 +30,83 @@
 using namespace std;
 using namespace sipixelobjects;
 
-
-SiPixelFedCablingMapBuilder::SiPixelFedCablingMapBuilder(const string & associatorName) : theAssociatorName(associatorName)
+//SiPixelFedCablingMapBuilder::SiPixelFedCablingMapBuilder(const string & associatorName) : theAssociatorName(associatorName)
+SiPixelFedCablingMapBuilder::SiPixelFedCablingMapBuilder(const string fileName, 
+							 const bool phase1) : 
+  fileName_(fileName), phase1_(phase1)
 { }
 
 
-SiPixelFedCablingTree * SiPixelFedCablingMapBuilder::produce( const edm::EventSetup& setup)
-{
-  FEDNumbering fednum;
+SiPixelFedCablingTree * SiPixelFedCablingMapBuilder::produce( const edm::EventSetup& setup) {
+
+  //bool phase1_ = false;  
+  //bool phase1_ = true;  
+
+  cout<<" buildr "<<phase1_<<endl;
+
+  int MINSiPixelFEDID = 0;
+  int MAXSiPixelFEDID = 39;
+  if(phase1_) {
+    // bpix 1200-1239, fpix 1240-1255
+    MINSiPixelFEDID = 1200;
+    MAXSiPixelFEDID = 1255;
+  }
+
+  //FEDNumbering fednum;
+  // fed range should come from input ot automatic from geometry
```

Comment:
As pointed out by @mmusich it seems that the range [1200-1349] is taken by `SiPixeluTCA`:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/DataFormats/FEDRawData/interface/FEDNumbering.h#L103-L104
Can you use them, or can you re-shuffle the ranges in `FEDNumbering`?